src/grow_worker/worker.py

```diff
@@ -23,6 +26,16 @@
 GROW_TASK_TYPE = GrowTaskType(os.environ.get("GROW_TASK_TYPE"))
 
 
+class EarlySystemExit(Exception):
```

Comment:
Moved from omotes-sdk-python to here. As Celery uses the SystemExit mechanism to also stop the running worker gracefully, before it would catch the SystemExit and not let the worker process terminate as expected.